DOCS-2392 Update SCALE Dashboard docs (#1003)
truenas/documentation: content/SCALE/GettingStarted/FirstTimeLogin.md

weight: 14
---

Now that you have installed and configured TrueNAS SCALE, you can log in to the web interface and begin managing data!

{{< expand "Can I configure TrueNAS SCALE using a CLI?" "v" >}}
After installing TrueNAS, you can configure and use the system through the web interface.

{{< hint warning >}}
**Important:** Use only the web interface to make configuration changes to the system.
{{< /hint >}}

By default, using the command-line interface (CLI) to modify the system **does not modify the settings database**.

> If users make any changes in the command line, the sytem will revert to the original database settings when it restarts.

Reviewer: ```suggestion
The system reverts to the original database settings when it restarts and wipes any user-made command line changes.
```

{{< /expand >}}

## Web Interface Access

By default, fresh installs of TrueNAS SCALE provide a default address for logging in to the web interface.
To view the web interface IP address or reconfigure web interface access, connect a monitor and keyboard to your TrueNAS system or connect with IPMI for out-of-band system management.

When powering on a TrueNAS system, the system attempts to connect to a DHCP server from all live interfaces and provide access to the web interface.
On networks that support Multicast Domain Name Services (mDNS), the system can use a hostname and domain to access the TrueNAS web interface.
By default, TrueNAS uses the hostname and domain *truenas.local*.
To change the hostname and domain in the web interface, go to **Network** and click *Settings* in the *Global Configuration* window.

To access the web interface using an IP address, use the one that the Console Setup Menu generated after installing SCALE, or use the one you configured in the [Post-install Configuration article]({{< relref "Post-installConfiguration.md" >}}) if you upgraded from CORE.

> If you need to, you can reset the root password in the TrueNAS console setup menu, or in the web interface by going to **Credentials > Local Users** and editing the `root` user.

Reviewer: ```suggestion
A strong login password is recommended!
You can reset the root password in the TrueNAS console setup menu or in the web interface by going to **Credentials > Local Users** and editing the `root` user.
```
